generated/nifake/custom_struct.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, data=None, struct_int=0, struct_double=0.0):
             self.struct_double = struct_double
 
     def __repr__(self):
-        return '{0}(data=None, struct_int={1}, struct_double={2}'.format(self.__class__.__name__, self.struct_int, self.struct_double)
+        return '{0}(data=None, struct_int={1}, struct_double={2})'.format(self.__class__.__name__, self.struct_int, self.struct_double)
```

Comment:
I'm guessing this was just a bug in unrelated, unexercised code?

Reply:
Yes, though it is cosmetic rather and doesn't cause an error.